server/server.js

```diff
@@ -1,25 +1,110 @@
 const express = require('express');
+
 const path = require('path');
+
 const app = express();
-const port = 8080;
+const port = 8070;
+const mysql = require('mysql');
+// const alert = require('alert');
+
+const bodyParser = require('body-parser');
+
+// const { check } = require('express-validator');
+
+app.use(bodyParser.urlencoded({ extended: false }));
 
 // Files stored statically in public folder
-app.use(express.static(path.join(__dirname,'../public/')))
+app.use(express.static(path.join(__dirname, '../public/')));
 
 app.post('/play', (req, res) => {
-    // TODO Check if guest name set, or logged in
+// TODO Check if guest name set, or logged in
 
-    res.sendFile(path.join(__dirname,'../public/play.html'))
-})
+  res.sendFile(path.join(__dirname, '../public/play.html'));
+});
 
-app.post('/login', (req, res) => {
-    // TODO Login existing user if exists, refuse otherwise
-})
+// following block of code creates the database and its tables locally.
+// Must do it outside post to avoid overwrite.
+const con = mysql.createConnection({
+  host: 'localhost', // use your own hostname
+  user: 'root', // use your mysql username
+  password: 'Aceg0864', // use your password
```

Comment:
I don't think this password should be in the remote repository, there are probably standard ways to best handle this we can look into (I've not done it before, but came across similar situations). I hope this isn't a password you use elsewhere as it's now publicly compromised.